scss/custom/_mixins.scss

```diff
@@ -0,0 +1,69 @@
+@mixin heading1() {
+  font-family: $font-family-base;
+  font-size: 48px;
```

Comment:
can you please make variables for the font sizes values as well? you can use something like xs, sm, md, lg, xl, etc. and then add them here! 

Something like: `$font-size-md: 20px`

Reply:
Done! `$font-size-md` and similar were already variables used thru Bootstrap, so I removed the first hyphens to make it like: `$fontsize-md`